### DISCUSSION TITLE
ftpのURLをhttpのURLにご変更

### DISCUSSION THREAD

**Discussion by timtak** [ffftp/ffftp#309] [Ideas]
FFFTPはどうもありがとうございます。
HPにアップするために使ってまいりました。

FTPのURLにpublic_htmlがございます場合（など）
ftpのURLをhttpのURLにご変更いただける機能を
お加えになっていただければありがたいです。

いつも
ftp://ftp.nihonbunka.com/public_html/temp/ff91.jpg
のようなコピーいただいたURLを
http://nihonbunka.com/temp/ff91.jpg
に変更しておりますが、間違ったりしみあすので、
自動化の設定がございましたら
便利だと存じます。

つまり
含まれるトリッガー: public_html
検索: ftp://ftp.nihonbunka.com/public_html
置換: http://nihonbunka.com

武本Timothy

**Comment by sayurin** [comment]
提案ありがとうございます。
正直、突拍子もないためあまり乗り気ではありません。
FTPサーバー名とHTTPサーバー名やFTP上のPathとHTTP上のPathと、対応関係が明確でないため、うまく対応できる場合とできない場合が生じます。うまく対応できた場合は実用的となりますが、それ以外のケースでは役に立たないという思いです。
